conda/dev.yml

```
@@ -8,12 +8,12 @@ dependencies:
   # =================
   - python >=3.9
   - pip
-  - matplotlib-base
-  - mpas_tools
+  - matplotlib-base # TODO: Find constraint
```

If you know nothing else, I would use the versions pinned in E3SM-Unified as the lower bounds and not have an upper bound.  However, with `matplotlib`, it might be best to have an upper bound for safety, since there are frequent, breaking changes:
https://github.com/MPAS-Dev/geometric_features/blob/93f71ba01d0722de7bfb1b9ec3e9129b859b8963/feature_creation_scripts/ismip6_landice_basins/basin_masks_to_geojson_regions.py#L116
```suggestion
  - matplotlib-base >=3.8.2,<3.10
```
The current version is 3.9.2 so this would be allowed, but 3.10 is under development and we could test it out before officially loosening the constraints to allow it in a future release.